esp-hal-common/src/mcpwm/operator.rs

```diff
@@ -1,3 +1,15 @@
+//! MCPWM peripheral - operator module
+//!
+//! ## Overview
+//! The `operator` module is part of the `MCPWM (Motor Control Pulse Width
```

Comment:
About the resolution of the abbreviation here, I think it is enough to resolve it in the parent module.